x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -31,7 +31,7 @@
 public class XPackLicenseState {
 
     public static final Set<OperationMode> FIPS_ALLOWED_LICENSE_OPERATION_MODES =
```

Comment:
Sounds like a reasonable change to me. There are a few other `public static boolean isXxxAllowedForOperationMode` methods. This can be another addition.

Reply:
[Update](https://github.com/elastic/elasticsearch/pull/51864/files#diff-75ffc27a1f769f6d57256b3a4f04babcR567)